CMakeLists.txt

```diff
@@ -112,6 +112,14 @@ add_definitions("-DENABLE_TBB")
 add_definitions("-DHAVE_TBB")
 add_definitions("-DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1")
 
+# Dwarf Bench
+option(ENABLE_DWARF_BENCH "Enable DwarfBench library for Cost Model" OFF)
+if(ENABLE_DWARF_BENCH)
+  find_package(dbench REQUIRED)
+  message("Dwarf Bench enabled")
```

Comment:
Such messages usually go to the `STATUS` channel. I'd also print a path to the found libraries in addition so that the message is more informative.